docs/messages_guide/src/asciidoc/_chapters/ddl_msgs.adoc

```diff
@@ -761,40 +814,42 @@ constraint specified in the CREATE INDEX statement.
 change the rows in the table to remove duplicates. Then reissue the
 statement.
 
-<<<
-[[SQL-1057]]
-== SQL 1057
+[[SQL-1055]]
+== SQL 1055
 
 ```
-{project-name} objects cannot be created on <volume-name>: File system error <error-number>.
+Object <privilege-object> already exists.
 ```
 
-Where <volume-name> is the name of a volume.
+Where <privilege-object> is an object you specified on a REGISTER statement.
 
-Where <error-number> is the file system error number.
+*Cause:* You attempted to register a new privilege object in the 
+{project-name} database but this object already exists.
 
-*Cause:* An attempt to create a file on volume <volume-name> resulted in
-file system error <error-number>.
+*Effect:* The operation takes no action.
 
-*Effect:* The operation fails.
-
-*Recovery:* For information about file system errors, see <<file_system_errors,File-System Errors>>.
+*Recovery:* If the object name is incorrect, correct and resubmit.
 
+<<<
 [[SQL-1058]]
 == SQL 1058
 
 ```
-Lock <lock-name> already exists.
+Unable to create histogram table <table-name>.
 ```
 
-Where <lock-name> is the name of the DDL lock.
+Where <table-name> is the name of the histogram table that could not be created.
 
-*Cause:* The statement required creation of a lock with a unique lock
-name, which was not generated.
+*Cause:* You attempted to execute a CREATE SCHEMA statement, but one
+of the histogram tables needed for the schema could not be created.
+This message may be accompanied by other messages that shed more light
+on the problem.
```

Comment:
Consistency:  in a previous message (1034) you stated "Additional error messages may give insight to the cause. " Here you state "This message may be accompanied by other messages that shed more light on the problem." Probably should say the same thing ...